master/Dockerfile

```diff
@@ -1,11 +1,14 @@
-FROM golang:1.17.1
+FROM python:3.7-alpine
 
-WORKDIR /app
+ENV FLASK_APP server.py
 
-COPY . .
+EXPOSE 8080
```

Comment:
nit: No need for that (docker-compose does the job). I'll drop in a follow-up commit.